.github/workflows/workflow.yml

```diff
@@ -36,9 +36,12 @@ jobs:
         with:
           python-version: '${{ matrix.python-version }}'
       - name: Install dependencies
-        run: pip install tox==3.25.1
+        run: |
+          pip install tox==3.25.1
+          pip install poetry
       - name: 'Run check ${{ matrix.tox-job }}'
         run: |
+          poetry update --lock
```

Comment:
please, dont do lock in CI, it's a long time operation
and actually should be done in repo and commited